$NetBSD: patch-src_crssync_CMakeLists.txt,v 1.3 2020/06/21 13:51:49 gdt Exp $

Fix executing dynamically-linked executable from the build directory

\todo Explain if this should be filed upstream or not.

--- src/crssync/CMakeLists.txt.orig	2020-06-19 12:10:56.000000000 +0000
+++ src/crssync/CMakeLists.txt
@@ -33,7 +33,7 @@ ELSEIF(CMAKE_CROSSCOMPILING)
   )
 ELSE()
 	ADD_CUSTOM_TARGET(synccrsdb
-		COMMAND ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/crssync
+		COMMAND env LD_LIBRARY_PATH=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/../lib ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/crssync
 		DEPENDS crssync
 	)
 ENDIF()
